Terran-One/cw-simulate-ui
# Standardized Address Component

## Issue link
https://trello.com/c/uaTxD8A4/97-add-copy-button-to-shortened-addresses

## Description
Introduced new `<Address />` component which standardizes how we display addresses in the app, w/ copy icon button, tooltip & `long` option

Also renamed `T1Popover` to more appropriate name `T1Info`. Removed `T1Info` from address as it was too large for summary tab, leaving less room for funds.

## Test steps
1. Create new simulation & instantiate contract
2. New address component should be found twice in Simulation screen, Accounts
Note that instances drawer menu does not use it as it uses different logic
